src/styles/index.scss

```diff
@@ -1,3 +1,97 @@
+$color-title: #060b35;
+$color-code: #616070;
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: red;
+}
+
+.card {
+  display: flex;
+  min-height: 100vh;
+  justify-content: center;
+  align-items: center;
+  background-color: blue;
+
+  &__block {
+    width: 200px;
+    height: 408px;
+    background-color: white;
+    box-sizing: border-box;
```

Comment:
`box-sizing: border-box;` is duplicated, so make sense move it to the `*{ box-sizing: border-box }`